<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>服务端</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="display">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>591</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperator">
    <property name="title">
     <string>服务端操作(&amp;S)</string>
    </property>
    <addaction name="actionStart_server"/>
    <addaction name="actionClose_server"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuInquire">
    <property name="title">
     <string>所有服务(&amp;Q)</string>
    </property>
    <addaction name="actionQuire_all_fights"/>
    <addaction name="actionQuire_fights"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
   </widget>
   <addaction name="menuOperator"/>
   <addaction name="menuInquire"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionStart_server">
   <property name="text">
    <string>开启服务器</string>
   </property>
  </action>
  <action name="actionClose_server">
   <property name="text">
    <string>断开服务</string>
   </property>
  </action>
  <action name="actionQuire_all_fights">
   <property name="text">
    <string>查询全部机票</string>
   </property>
  </action>
  <action name="actionQuire_fights">
   <property name="text">
    <string>指定查询机票</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
